tensorboard/webapp/_angular_material_theming.scss

```diff
@@ -0,0 +1,16 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+@import '@angular/material/theming';
```

Comment:
This shim looks a little surprising. Can you add a comment to the file to explain why this is necessary, i.e., why the files that import this file can't just import @angular/material/theming directly?

Reply:
I deliberately avoided leaving comment in this code. This is required for sync basically (internal dependencies are a lot more granular than just _theming.scss in OSS).

Reply:
Understood.